samples/e2eTestUtils/ElectronPlaywrightTestUtils.ts

```diff
@@ -112,3 +112,16 @@ export function createFolder(foldername: string) {
         fs.mkdirSync(foldername, { recursive: true });
     }
 }
+
+export async function getAuthCodeUrl(page: Page): Promise<string> {
+    const msgPromise = await page.waitForEvent("console", {
+        predicate: async (message) => {
+            const text = message.text();
+            if (text.includes("https://login.microsoftonline.com")) {
```

Comment:
Better would be the text starts with the string "https://login.microsoftonline.com/"  Adding the trailing slash is a good practice as it prevents domains from matching that look like https://login.microsoftonline.com.attackerdomain.com/"

Reply:
Done